### ISSUE TITLE
Able to Connect (Sorta)

### ISSUE THREAD

**Issue by skidda666** [custom-components/remote_homeassistant#295]
I've successfully installed an instance of remoteHA on both my home HA and my office HA, I used my home HA as my main node and the office as the remote node. I am also using Cloudflare Tunnel that points to my http://localhost:8123 on my remote instance. When I go to my remote home assistance's site it's up and actively running (https://homeassistant.mysite.com), i'm able to sign in and navigate. However when I do it through the remote HA instance it's throwing back a 400 error an http port was redirected to an https port and throws back the site and it's port (http://homeassistant.mysite.com:443). Where cloudflare is already negotiating the port on it's end is it really necessary to put a required port in the setup field? I've been fighting with this for the past few days, any help or suggestions would be greatly appreciated.

**Comment by jaym25** [comment]
@skidda666 
Yes.  You need to set the port to 443 on the Remote HA main instance and check Secure and Verify SSL when setting it up.  If using yaml, the first lines should look like this:

remote_homeassistant:
  instances:
  - host: homeassistant.mysite.com
    port: 443
    secure: true
    verify_ssl: true
